latch_cli/constants.py

```diff
@@ -4,6 +4,13 @@
 from dataclasses import dataclass
 
 
+@dataclass
+class LatchConstants:
+    base_image: str = (
```

Comment:
lovely, do u mind moving all the other scattered constants here too

Reply:
also make the configs singletons

Reply:
Sure, sure